Misc/NEWS.d/next/Library/2019-09-12-12-11-05.bpo-25597.mPMzVx.rst

```diff
@@ -0,0 +1,3 @@
+Ensure ``wraps`` is supplied to :class:`unittest.mock.MagicMock` is used to
+calculate return values for the magic methods instead of using the default
```

Comment:
"Ensure, if ``wraps`` is supplied to :class:`unittest.mock.MagicMock`, it is used to" maybe?

Reply:
Thanks Chris, updated.